### ISSUE TITLE
pycodestyle 2.1.0 incorrectly flags E305 on module-level constants

### ISSUE THREAD

**Issue by sirosen** [PyCQA/pycodestyle#601]
Our build server pulled in the latest this morning, and we saw failures on files which should be valid because of this.

pycodestyle 2.1.0 seems to be matching for E305 on lines with `def` or `class` as a prefix, when it should be looking for `def ` and `class `.

Simple example:
```python
classic_value = 0
default_foo = 1
default_bar = 2
```

Those second two lines will be flagged for E305 because there isn't enough whitepace after `classic_value` or `default_foo`.

Fix should be simple -- make sure that match requires whitespace -- and this needs a testcase too. I'll be fixing my own code to get past the slightly broken validation, then look into submitting a PR if I can figure out this codebase (never made a contribution before).
As always, thanks for great tools!

EDIT: This can be considered a duplicate of #593 . Sorry for not finding that before submitting.

**Comment by sirosen** [comment]
I just checked and this seems to be fixed in `pycodestyle==2.2.0.dev0`.

Should this be closed? I still have a test case I can add for posterity, if that's worth it to anyone.

**Comment by IanLee1521** [comment]
Let's go ahead and close this since it's fixed. Based some some comments, I think 2.2.0 is going to end up getting released shortly, which will address this.

**Comment by IanLee1521** [comment]
Version 2.2.0 was just released, so this should be fixed now.
